src/store/modules/page.js

```diff
@@ -15,7 +15,7 @@ const getters = {
   page: state => id => {
     let field = typeof id === 'number' ? 'id' : 'slug';
     let page = state.all.filter(page => page[field] === id);
-    return (page[0]) ? page : false;
+    return (page) ? page[0] : false;
```

Comment:
This condition will always be true. You should keep the condition page[0] but return page[0].
You can prove this to yourself by pasting the following into chrome/firefox console:

```
var x = [] ? true : false
console.log(x) // will be true
```

Reply:
Thank you! I have committed the fix you suggested